### DISCUSSION TITLE
Browser tabs

### DISCUSSION THREAD

**Discussion by br3eze** [davatorium/rofi#1607] [Ideas/Feature Requests]
Hello.

Can I use rofi to search through a browser, or even browsers, tabs? 
KDE implements this for example for the Firefox, at least as I know.

And thanks for the rofi, great tool! )

**Comment by DaveDavenport** [comment]
I don´t know, there might exists a script/plugin that implements this.

**Comment by vredesbyyrd** [comment]
Late answer - but you very easily accomplish that leveraging [brotab](https://github.com/balta2ar/brotab). 

With `brotab` its as simple as `bt list | rofi -dmenu`. Then parse the output in anyway you like. I believe there are some rofi/fzf examples in the brotab repo. 

I got carried away with [rofi-blocks](https://github.com/OmarCastro/rofi-blocks) + brotab/simple tab groups + history/suggestions. First it searches tabs, if no matches fall back to a combined view of duckduckgo search suggestions and browser history.  

My point is, you can get as fancy (or crazy) as you like.
